tests/test_images.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
heh - will you be updating all scripts you touch? I think all our scripts are hard wired to /bin/bash - so, that could keep you busy for some time ;-)

Reply:
That's not so bad if you just do that when stumbling on it :)
Btw I started to care about this when reviewing kata-containers/tests#692